Makefile

```diff
@@ -54,35 +54,37 @@ create-feedback-table:
 
 # (Re)Generate config listing for a service type
 service-type-config-pg:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "pg" > includes/config-pg.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "pg" "includes/config-pg.rst"
 
 service-type-config-kafka:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "kafka" > includes/config-kafka.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "kafka" "includes/config-kafka.rst"
 
 service-type-config-cassandra:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "cassandra" > includes/config-cassandra.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "cassandra" "includes/config-cassandra.rst"
 
 service-type-config-opensearch:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "opensearch" > includes/config-opensearch.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "opensearch" "includes/config-opensearch.rst"
 
 service-type-config-mysql:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "mysql" > includes/config-mysql.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "mysql" "includes/config-mysql.rst"
 
 service-type-config-m3db:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "m3db" > includes/config-m3db.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "m3db" "includes/config-m3db.rst"
 
 service-type-config-redis:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "redis" > includes/config-redis.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "redis" "includes/config-redis.rst"
 
 service-type-config-flink:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "flink" > includes/config-flink.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "flink" "includes/config-flink.rst"
 
 service-type-config-grafana:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "grafana" > includes/config-grafana.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "grafana" "includes/config-grafana.rst"
 
 service-type-config-influxdb:
-	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "influxdb" > includes/config-influxdb.rst
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config.py" "influxdb" "includes/config-influxdb.rst"
 
+service-type-config-all:
+	python "$(SOURCEDIR)/scripts/aiven/service_type_config_all.py"
```

Comment:
I wonder if this is really needed, we could after all just use make and use the other commands as dependencies

Reply:
not needed, fixed